relevanceai/operations/cluster/cluster.py

```diff
@@ -628,3 +637,196 @@ def aggregate(
+        measurement_name: str = "measurement",
+    ):
+        """
+        Plot the distributions across each cluster
+        measure_function is run on each cluster and plots
+
+        Example
+        --------
+        .. code-block::
+            from scipy.stats import skew
+            ops.plot_distributions_measure(numeric_field, skew, dataset_id=dataset_id)
+        """
+        try:
+            import seaborn as sns
+            import matplotlib.pyplot as plt
+        except ModuleNotFoundError:
+            print("You need to install seaborn! `pip install seaborn`.")
+        cluster_field = self._get_cluster_field_name()
+
+        # use the max and min to make the x axis the same
+        if dataset_id is None:
+            dataset_id = self.dataset_id
+        numeric_field_facet = self.datasets.facets(
+            dataset_id=dataset_id, fields=[numeric_field]
+        )
+
+        facet_result = numeric_field_facet["results"][numeric_field]
+
+        docs = self._get_all_documents(
+            dataset_id=dataset_id if dataset_id is None else dataset_id,
+            select_fields=[numeric_field, cluster_field],
+        )
+        df = pd.json_normalize(docs)
+        top_comms = df[cluster_field].value_counts()
+        cluster_measurements = {}
+        for community in tqdm(top_comms.index):
+            sample_comm_df = df[df[cluster_field] == community]
+            measure_output = measure_function(
+                sample_comm_df[numeric_field].dropna().to_list()
+            )
+            cluster_measurements[community] = measure_output
+
+        cluster_measurements = {
+            k: v
+            for k, v in sorted(
+                cluster_measurements.items(), key=lambda item: item[1], reverse=asc
+            )
+        }
+
+        for i, (community, measurement) in enumerate(cluster_measurements.items()):
+            if i == top_indices:
+                return
+            sample_comm_df = df[df[cluster_field] == community]
+            g = sns.displot(
+                sample_comm_df[numeric_field],
+            )
+            g.set(xlim=(facet_result["min"], facet_result["max"]))
+            plt.title(community + str(f" - {measurement_name}: {measurement}"))
+
+    def plot_most_skewed(
+        self,
+        numeric_field: str,
+        top_indices: int = 10,
+        dataset_id: str = None,
+        asc: bool = True,
+    ):
+        """
+        Plot the skewness of your clusters.
+        """
+        from scipy.stats import skew
+
+        return self.plot_distributions_by_measure(
+            numeric_field=numeric_field,
+            measure_function=skew,
+            top_indices=top_indices,
+            dataset_id=dataset_id,
+            asc=asc,
+        )
+
+    def _check_for_dataset_id(self):
+        if not hasattr(self, "dataset_id"):
+            raise ValueError(
+                "You are missing a dataset ID. Please set using the argument dataset_id='...'."
```

Comment:
Message should in `relevanceai.constants.messages`

Reply:
Moved to `relevanceai.constants.errors`